start/manage/fleets/edit.php

```diff
@@ -0,0 +1,74 @@
+<?php
```

Comment:
The name of this file suggests that you are editing the Fleet. I think a better name would be `editShipFleet.php`